src/components/horizonCard/HorizonCard.ts

```diff
@@ -41,6 +41,28 @@ export class HorizonCard extends LitElement {
     this.processLastHass()
   }
 
+  /**
+   * called by HASS to properly distribute card in lovelace view. It should return height
+   * of the card as a number where 1 is equivalent of 50 pixels.
+   * @see https://developers.home-assistant.io/docs/frontend/custom-ui/custom-card/#api
+   */
+  public getCardSize (): number {
+		let height = 4 // Smallest possible card (only graph) is roughly 200px
+
+    // Each element of card (header, content, footer) adds roughly 50px to the height
+    if(this.config.fields?.sunrise || this.config.fields?.sunset) {
```

Comment:
Please put a space after the ifs to make it consistent with the rest of the code.

Reply:
Sorry for nitpicking but this isn't resolved.

Reply:
I thought you meant spaces between if conditions not after actual statements, sorry!